.travis.yml

```diff
@@ -10,7 +10,7 @@ cache:
     - '$HOME/.sonar/cache'
 env:
   # This configures the three separate jobs to run
-  - COV_FLAG="unit"        SONAR="true"  SCRIPT="./mvnw verify --batch-mode --quiet -Dmaven.test.redirectTestOutputToFile=true -Dskip.js.build=true -DskipITs"
+  - COV_FLAG="unit"        SONAR="true"  SCRIPT="./mvnw findbugs:check verify --batch-mode --quiet -Dmaven.test.redirectTestOutputToFile=true -Dskip.js.build=true -DskipITs"
```

Comment:
Did you check how the 'unit' build time changed (does it complete before all builds complete)?

Reply:
No, this was just to see it work. Moving it to a separate job.